wikipedia_scraper/pipelines.py

```diff
@@ -6,7 +6,65 @@
 
 # useful for handling different item types with a single interface
 from itemadapter import ItemAdapter
+from scrapy.exceptions import DropItem
+import pymongo
+import json
 
+class MongoDBPipeline(object):
```

Comment:
I upgraded this code slightly and put it in main so we can try using that version (it handles duplicate records, allows for the use of a login and password, ect.)